### ISSUE TITLE
Desktop app's new tab focus order creates usability issues

### ISSUE THREAD

**Issue by aiuax** [bitwarden/desktop#1325]
### Steps To Reproduce

1. Open Bitwarden desktop app for Windows (v1.31.0)
2. Enter master password
3. Once unlocked, cursor is appropriately placed on 'Search Vault' text entry field.
4. Press `TAB` three times
5. Note that element focus does not move to the item selection panel but simply moves down the category/folder list

### Expected Result

Previously, tab order was set such that pressing tab three times would navigate to the item selection panel, allowing quick selection of items for automatic entry.

In 1.31.0, this is now impossible to achieve and makes keyboard navigation infeasible.

### Actual Result

Attempting to use the `TAB` key to navigate to a searched entry is unwieldy, as it now requires dozens of presses to navigate to the item selection panel.  Pressing `TAB` several times when the search text entry field is focused instead navigates down the left (category/folder) column.

### Additional Context

This is a reversion in terms of user experience.  Keyboard navigation has become less friendly and interacting with individual entries has become extremely difficult.

### Operating System

Windows

### Installation method

Direct Download (from bitwarden.com)

### Build Version

1.31.0

**Comment by keywordnew** [comment]
+1 to this. My experiencing a variation of the original issue, but these are essentially the same. I account for the difference below with only those sections that are different.

### Steps To Reproduce

1. Navigate to search field.
2. Make search to filter entry list
3. Press `tab` once
4. Focus moves to the new user selection dropdown
5. Press `tab` once
6. Focus moves to the sidebar
7. Press `tab` again n times
6. Focus moves down the sidebar

### Expected Result

1. Navigate to search field.
2. Make search to filter entry list
3. Press `tab` once
4. Focus changes to the first entry of the filtered entry list
5. Press `tab` n times
6. Focus moves down the filtered entry list

### Operating System

MacOS

### Build Version

1.31.2

**Comment by allparadigm** [comment]
Confirm affects Flatpak on linux as well as Windows

No longer any efficient way to navigate from keyboard

**Comment by Hinton** [comment]
Hi,

Thanks for providing this feedback, we are discussing internally what the preferred tab order should be.

**Comment by addisonbeck** [comment]
Thanks for the report, y'all. We're on this and will have a patch release with a fix out ASAP.

**Comment by keywordnew** [comment]
With a predictable double tap on `tab`, this is still hugely preferable to the current state reported in this issue, thank you for merging a fix.

#1140 makes it so that the account switcher still comes before the item list in the focus order, is that correct? Curious, the design choice suggests the account switcher is expected to be used after a search action more frequently than the item list.

**Comment by danielleflinn** [comment]
@keywordnew You are correct; the focus order does include the account switcher prior to search results. This was a technical restriction to get a quick fix out for this bug, and give the team more time to further improve the Desktop's keyboard navigation. 

We are exploring ways to allow users to navigate the filter and item list using arrow keys. I've attached a proposed long term solution to focus order. Let me know if you have any feedback on the order shown. 

![Focus order](https://user-images.githubusercontent.com/43477473/154128726-b32adb68-964f-4168-87d1-d7626963c8a2.png)

**Comment by patrickhlauke** [comment]
although this is already closed, just dropping in the idea here of using `F6` to allow jumping directly between different logical parts/panels https://github.com/bitwarden/desktop/issues/1332#issuecomment-1040705036

**Comment by keywordnew** [comment]
@danielleflinn I've annotated your visual. Thank you for providing the artifact, much easier to share the feedback solicited.

![154128726-b32adb68-964f-4168-87d1-d7626963c8a2](https://user-images.githubusercontent.com/10335475/154856825-94259887-dfe0-4abf-ab09-d621f9829f59.png)

* I made one modification to the tab order－specifically, switching 2 and 3. Interacting with search results is the _likely_ next step to _most_ searches.
* It doesn't seem necessary (to me) that the tab order cycles through to 7 (Item information). Interacting with item information typically follows the user selecting an item in 2, the item list. If so, interacting with an item's information would follow pressing `Return` to select that item from the item list. Thus, the label 2a.
* Adding the left/right arrows (to up/down) as alternative keys for navigating lists gives folks flexibility, especially if they customize keyboard layouts for ergonomics.

_Thanks for the great work done by you and your team. This tab order thing aside, I suspect I've already benefited from your and their efforts._
